### ISSUE TITLE
Failed to call refresh

### ISSUE THREAD

**Issue by ezbik** [plathrop/puppet-module-supervisor#83]
Puppet agent output:

```
Info: Computing checksum on file /etc/hhvm/php.ini
Info: FileBucket got a duplicate file {md5}8cd4a443aee933bc90f530c642bd6b4c
Info: /Stage[main]/Hhvm/File[/etc/hhvm/php.ini]: Filebucketed /etc/hhvm/php.ini to puppet with sum 8cd4a443aee933bc90f530c642bd6b4c
Notice: /Stage[main]/Hhvm/File[/etc/hhvm/php.ini]/content: content changed '{md5}8cd4a443aee933bc90f530c642bd6b4c' to '{md5}32d5b5ba1b9b64fc08f9bbe980df17cd'
Info: /Stage[main]/Hhvm/File[/etc/hhvm/php.ini]: Scheduling refresh of Supervisor::Service[hhvm]
Notice: /Stage[main]/Apt::Update/Exec[apt_update]/returns: executed successfully
Notice: /Stage[main]/Software::Basic/Exec[apt-get update]/returns: executed successfully
Info: Supervisor::Service[hhvm]: Scheduling refresh of Service[supervisor::hhvm]
Error: /Stage[main]/Hhvm/Supervisor::Service[hhvm]/Service[supervisor::hhvm]: Failed to call refresh: undefined method `empty?' for nil:NilClass
Error: /Stage[main]/Hhvm/Supervisor::Service[hhvm]/Service[supervisor::hhvm]: undefined method `empty?' for nil:NilClass
Notice: Finished catalog run in 23.36 seconds
```

Puppet manifest:

```
class { 'supervisor':
conf_dir => '/etc/supervisor/conf.d',
  conf_ext => '.conf',
}

supervisor::service {
  'hhvm':
    ensure      => present,
    command     => '/usr/local/bin/hhvm.wrapper',
    subscribe     => [ File["/etc/hhvm/server.ini"],
                    File["/etc/hhvm/php.ini"],
                    File["/usr/local/bin/hhvm.wrapper"],
                    Package["hhvm"],
            ],
}
```

Software versions:

```
ii  facter                           2.3.0-1puppetlabs1                Ruby module for collecting simple facts about a host operating system
ii  hiera                            1.3.4-1puppetlabs1                A simple pluggable Hierarchical Database.
ii  libaugeas-ruby1.8                0.3.0-1.1ubuntu4                  Augeas bindings for the Ruby language
ii  libjson-ruby                     1.6.3-1                           Transitional package for ruby-json
ii  libruby                          4.8                               Transitional package for libruby1.8
ii  libruby1.8                       1.8.7.352-2ubuntu1.6              Libraries necessary to run Ruby 1.8
ii  libruby1.9.1                     1.9.3.0-1ubuntu1                  Libraries necessary to run Ruby 1.9.1
ii  libshadow-ruby1.8                1.4.1-8build1                     Interface of shadow password for Ruby 1.8
ii  puppet                           3.7.3-1puppetlabs1                Centralized configuration management - agent startup and compatibility scripts
ii  puppet-common                    3.7.3-1puppetlabs1                Centralized configuration management
ii  ruby                             4.8                               Transitional package for ruby1.8
ii  ruby-json                        1.6.3-1                           JSON library for Ruby
ii  ruby1.8                          1.8.7.352-2ubuntu1.6              Interpreter of object-oriented scripting language Ruby 1.8
ii  ruby1.9.1                        1.9.3.0-1ubuntu1                  Interpreter of object-oriented scripting language Ruby
ii  vim-puppet                       2.7.11-1ubuntu2.7                 syntax highlighting for puppet manifests in vim
```

**Comment by ezbik** [comment]
The issue self-resolved when I downgraded this puppet module from HEAD 16fc08c0   to [0.1.0 release] ( https://github.com/plathrop/puppet-module-supervisor/releases/download/v0.1.0/jimdo-supervisor-0.1.0.tar.gz)

**Comment by skrings** [comment]
@ezbik The `undefined method empty? for nil:NilClass` error should be fixed by #84. Could you test 4ceae890cdef607a78b933a62d10880ed2644765? I would appreciate your feedback on this

**Comment by plathrop** [comment]
Closing due to lack of response from reporter.
